### DISCUSSION TITLE
php tooling?

### DISCUSSION THREAD

**Discussion by bobrowski** [phcode-dev/phoenix#2016] [Q&A]
im wondering if php tooling is still a thing, or is it built in now? in old brackets i had this extra config:

// PHP Tooling default configuration settings
"php": {
        "enablePhpTooling": true, //false to disable PHP features
	"executablePath": "php"//Path format: "C:\\path\\to\\php.exe" for WIN  or "/Users/someuser/bin/php" for MAC & Linux
	"memoryLimit": "4095M", //Specify a memory limit for the PHP language server process
	"validateOnType": "false" //Configuration to have diagnostics "on type" or "on save"
}

thanks

**Comment by abose** [comment]
Thanks for reporting the issue. PHP tooling is not currently in phcode, mainly as we wanted to modernize the framework as the PHP tooling in brackets is more than 5 years old. This will be done later in mid 2025 and may not come very soon.
